nuxeo-java-client-test/src/test/java/org/nuxeo/client/ITDirectory.java

```diff
@@ -69,8 +69,10 @@ public void itCanCreateUpdateFetchDeleteDirectory() {
         entry.putIdProperty("test");
         entry.putLabelProperty("test");
         entry.putObsoleteProperty(0);
-        entry.putOrderingProperty(0);
+        entry.putOrderingProperty(0L);
         DirectoryEntry result = directory.createEntry(entry);
+        Long l = result.getOrderingProperty();
+        assertNotNull(l);
```

Comment:
`assertEquals` is more interesting than `assertNotNull` in case we know the value. 

`assertNotNull` is useful to check nullity of an object before calling method on it.
```suggestion
        assertEquals(0L, result.getOrderingProperty());
```
Furthermore, could this assertion be moved after the not null assertion made on the `result` object.